lib/v2/acm-ecnu/contest.js

```diff
@@ -0,0 +1,44 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { art } = require('@/utils/render');
+const path = require('path');
+
+module.exports = async (ctx) => {
+    const category = ctx.params.category || '';
```

Comment:
```suggestion
    const category = ctx.params.category ?? '';
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator